### PR TITLE
New bindings and samples for vSphere 7.0

### PR DESCRIPTION
**Pull request by anusha94**
Sanity testing done for the new samples

Signed-Off-By: Anusha Hegde <anushah@vmware.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,16 +214,17 @@ $ python samples/vsphere/vcenter/vm/list_vms.py -v
 ### vSphere API Documentation
 
 * [VMware Cloud on AWS vSphere (latest version)](https://vmware.github.io/vsphere-automation-sdk-python/vsphere/cloud/index.html)
-* [vSphere 6.7.1 (latest)](https://vmware.github.io/vsphere-automation-sdk-python/vsphere/6.7.1/)
```
Comment (kunal-pmj) [inline README.md:217]: This link should go in previous releases

File: lib/vapi-client-bindings/index.html [hunk 2]
```diff
@@ -1 +1 @@
-<a href='vapi_client_bindings-3.2.0-py2.py3-none-any.whl'>vapi_client_bindings-3.2.0-py2.py3-none-any.whl</a><br />
```
Comment (kunal-pmj) [inline lib/vapi-client-bindings/index.html:1]: remove trailing space

File: README.md [hunk 3]
```diff
@@ -240,8 +241,7 @@ Common issues you may run into while installing the sdk and running samples are
 Board members are volunteers from the SDK community and VMware staff members, board members are not held responsible for any issues which may occur from running of samples from this repository.
 
 Members:
-* Tianhao He (VMware)
```
Comment (kunal-pmj) [inline README.md:243]: Add a secondary Owner

File: lib/vapi-client-bindings/index.html (hunk 2 above)
Comment (anusha94) [reply lib/vapi-client-bindings/index.html:1]: This file is edited by the generator. However, I couldn't find the trailing space in the editor.

Comment (kunal-pmj) [reply lib/vapi-client-bindings/index.html:1]: yes, ignore it

**Pull request merged**